### DISCUSSION TITLE
Extension for Yandex.Games SDK

### DISCUSSION THREAD

**Discussion by timaaos** [TurboWarp/extensions#450] [Show and tell]
I have made an extension, that makes possible to publish scratch games and earn money on Yandex.Games. It uses Yandex.Games SDK for JavaScript. My extension is coded badly, but it is used in some games with big success: 
-  Bad example 🤮: https://yandex.com/games/app/223944 . 100K players played that "game". Developer made ~400$.
-  Better example 😐: https://yandex.ru/games/app/222429 . 5K players played that "game", developer made ~40$.
- Extension is new, so not many devs used, but i hope somebody will use for good quality games.

It is unsandboxed extension, because it adds <script> tag to HTML file, so extension can correctly work. 

thats the link to the extension on GitHub (only russian language of wiki and README because Yandex.Games is mostly Russian community):  https://github.com/timaaos/YandexGamesTurboWarp

Write your thoughts in that discussion.

**Comment by snowboyz0825** [comment]
I dunno what's on this website but i'm scared
